index.html

```diff
@@ -4350,7 +4350,7 @@ <h2>Secure Transport</h2>
           </dd>
           <dt>Private Networks:<dt>
           <dd><span class="rfc2119-assertion" 
-              id="arch-security-consideration-tls-mandatory">When a Thing is made 
+              id="arch-security-consideration-tls-mandatory-priv">When a Thing is made 
```

Comment:
actually, is a should assertion, so should be "arch-security-consideration-tls-recommended-priv"